src/ControllerMethod.php

```diff
@@ -230,6 +230,10 @@ public static function parse(string $context,
 				continue;
 			}
 
+			if (str_contains($param->type->description, '@deprecated')) {
+				$param->type->deprecated = true;
+			}
```

Comment:
I would suggest replacing `@deprecated` with `Deprecated:` in the description to make it a bit more readable like a normal sentence.

Reply:
But then we loose IDE highlighting:
![grafik](https://github.com/user-attachments/assets/6f863ac2-6be8-44b4-b2c7-f0e6603e44ee)

Reply:
Ah, in the parsed openapi.json
We can try that